discord-scripts/fix-linear-embed.ts

```diff
@@ -13,6 +13,34 @@ const { LINEAR_API_TOKEN } = process.env
 
 // track processed message to avoid duplicates if original message is edited
 const processedMessages = new Map<string, Set<string>>()
+// let us also track sent embeds to delete them if the original message is deleted or edited WIP
+const sentEmbeds = new Map<string, Message>()
+
+let issueTagRegex: RegExp | null = null
```

Comment:
If `createLinearEmbed` already handles the possibility where the issue URL couldn't be resolved, I wonder if we just make this, generically, `\W[A-Z]{3,}-[0-9]+`, rather than trying to resolve issue prefixes? The advantage here is that resolving issue prefixes implies we need to periodically refresh them.